### DISCUSSION TITLE
Harman - Advertising / Promoting AIR ?

### DISCUSSION THREAD

**Discussion by 2jfw** [airsdk/Adobe-Runtime-Support#1465] [Q&A]
@ajwfrost:
On the Starling forum the topic came up about Harman not actively _advertising_ / _promoting_ **AIR** - is there any reason for that you could tell us?
Just wondering what the plan is and where Harman/you will head to ...

The upcoming website https://airsdk.dev/ looks quite promising - is there any plan on making it aware to the broad public (e.g. when enough content is available or a specific mile stone has been reached)?
If so, how do you plan on reaching out the the people/public?

**Comment by Shaunmax** [comment]
I would like to know, how many developers are currently subscribed to Harman's paid AIR sdk program.

**Reply by raresn** [reply]
That is a very good question. I often wonder how big is our community :)

**Reply by ajwfrost** [reply]
Hi

The actual number of subscribers is someone "commercially confidential" but is enough for what we need, and encouragingly is continuously increasing. So the number of folk paying for the SDK is still growing - whether that is from people who were on 'free tier' licenses or whether it's folk coming from the Adobe versions - or from completely new-to-AIR folk - we don't really know.

We haven't done the analysis for a while now, but perhaps a year or so ago we had a quick look at the balance of 'paid for' vs 'free tier' licensees, and it was about 30:70 i.e. over twice as many people using it for free as the number who were subscribing. Whilst we're happy with the commercial viability of the AIR SDK, I am of course being tasked by my bosses to try to increase revenue, and one of the ways to do that is to 'encourage' folk from the free tier to the basic license payment - hence the Linux versions only being available to paid licensees, and we'll potentially have similar "premium" features where it makes sense.

I'll see if I can get a dump of data from the analytics system and have someone provide some more up to date figures. In case it's of interest: a few times recently I've had to do presentations/demos to other folk here, and I sometimes just log onto AWS and hook into the analytics system where, whenever you use ADT to package up an application, it sends us details of the operating system, IDE used, SDK version etc. And it's gratifyingly quick and frequent, every second or two someone is packaging up their application for IPAs, APKs, etc..

In terms of the original question about actively advertising and promoting AIR .. we are trying to get a few more things in order before we try reaching out to new folk:
a) the website: airsdk.dev being the better portal as it has the flexibility that we don't have when using the Harman domain (whicn is controlled by marketing teams etc). And of course, it's open to user contributions, so please do contribute!
b) the documentation: we are about to get permission to start hosting a lot more of the docs such as the AS3 APIs and reference guides etc, so this will also go over to airsdk.dev (at least once we've also worked out how to generate that handy html/js app that Adobe use for the AS3 documentation..)
c) the 'getting started' approach - some tweaks still needed on the website to the guidance documentation, but also we have been looking at how to make it easier for folk to obtain the SDK and keep it up to date, particularly on macOS. So there are a couple of initiatives that are ongoing that we want to get published first.

Once all that's done .. well yes, it would be great to promote it, maybe through some media blogs, advertising, and education (we still see a surprising number of folk using AS3 and Flash/AIR in college courses - good to hook them young!). Plus we're picking up some business from companies that had been using Flash Player and who are looking for relatively cheap ways to continue having their apps working without completely migrating off to JS etc..

Hope that helps... and it's always good to get suggestions and feedback, so please keep it coming :-)

thanks

**Reply by GeneralVimes** [reply]
This is great news, Andrew!
Besides making my own games I teach computer science and I also taught AS3 to my student to demonstrate how easy it is to make a game.

Also, this reminds me... I constantly forget to post my games for airsdk.dev showcase - will surely do to support the community!

**Reply by 2jfw** [reply]
@ajwfrost Thanks a lot for your comprehensive answer on this! Sounds great and I'm really looking forward to it!
I love the fact, that you _do_ actually have insight of the utilization of AIR SDK (ADT) - sounds very promising :)
